tests/services/validate_file_transfer_service/conftest.py

```diff
@@ -0,0 +1,105 @@
+from pathlib import Path
+
+import pytest
+
+from cg.models.cg_config import CGConfig
+from cg.services.validate_file_transfer_service.validate_file_transfer_service import (
+    ValidateFileTransferService,
+)
+from cg.services.validate_file_transfer_service.validate_pacbio_file_transfer_service import (
+    ValidatePacbioFileTransferService,
+)
+
+
+@pytest.fixture
+def manifest_file(fixtures_dir: Path) -> Path:
+    """Return the path to a manifest file."""
+    return Path(fixtures_dir, "services", "validate_file_transfer_service", "pacbio_transfer_done")
+
+
+@pytest.fixture
+def manifest_file_fail(fixtures_dir: Path) -> Path:
+    """Return the path to a manifest file."""
+    return Path(
+        fixtures_dir, "services", "validate_file_transfer_service", "pacbio_transfer_done_fail"
+    )
+
+
+@pytest.fixture
+def source_dir(tmp_path: Path) -> Path:
```

Comment:
There are no name clashes yet, but it seems like a good idea as `source_dir` is very general
```suggestion
def transfer_source_dir(tmp_path: Path) -> Path:
```